src/util/config/build.js

```diff
@@ -84,23 +28,6 @@ export const schema = {
 };
 export const config = convict(schema);
 
-// Load environment dependent configuration
-const configPath = path.resolve(
```

Comment:
I thought briefly about that, but was thinking that `disable_hmr` might generally be something you want to temporarily opt into rather than 'hard coding' it into a local config file, so maybe it would be simplest just to leave it as an env variable - the hoops we have to jump through to read a local config file are a little cumbersome for what IMO is an edge case usage. The original desire for `DISABLE_HMR` came from troubles with the old implementation of HMR, so I'm not sure if it will even be used these days...

I can add back support if it seems like it's too inconsistent or awkward to use.